server/src/main/scala/ch.epfl.scala.index.server/Server.scala

```diff
@@ -156,15 +164,15 @@ object Server {
             complete(artifactPage(reference, SemanticVersionParser(version), userState.map(_.user)))
           }
         } ~
-        path(Segment) { owner =>
-          parameters('artifact, 'version.?){ (artifact, version) =>
-            val rest = version match {
-              case Some(v) if !v.isEmpty => "/" + v
-              case _ => ""
-            }
-            redirect(s"/$owner/$artifact$rest", StatusCodes.PermanentRedirect)
-          }
-        } ~
+        // path(Segment) { owner =>
+        //   parameters('artifact, 'version.?){ (artifact, version) =>
```

Comment:
`artifact` was a required parameter here. Given the path, it doesn't make much sense like this.